projects/plugins/launchpad/composer.json

```diff
@@ -7,6 +7,7 @@
 		"automattic/jetpack-assets": "@dev",
 		"automattic/jetpack-autoloader": "@dev",
 		"automattic/jetpack-composer-plugin": "@dev",
+		"automattic/jetpack-config": "@dev",
```

Comment:
I'm curious on how this dependency didn't make it here even tho it's present in starter plugin. Any clue?

Reply:
I'm not sure I understand. Are you referring to the config package?

Reply:
Yes. Because the jetpack-config package is present in the starter plugin, so I don't understand why you needed to add it by hand

Reply:
Ah. I think I may have accidentally removed it in the original commit.